tests/v2/actions/charts/stackstate.go

```diff
@@ -26,12 +28,78 @@ const (
 	StackstateNamespace          = "stackstate"
 	StackstateCRD                = "observability.rancher.io.configuration"
 	RancherPartnerChartRepo      = "rancher-partner-charts"
+	StackStateServerChartRepo    = "suse-observability"
+	StackStateServerNamespace    = "suse-observability"
 )
 
 var (
 	timeoutSeconds = int64(defaults.TwoMinuteTimeout)
 )
 
+// InstallStackStateServerChart installs the StackState chart into the specified Kubernetes cluster and namespace.
+// It uses Rancher client and configuration details, and waits for the chart deployment to complete successfully.
+// Parameters:
+// - client: the Rancher client used for connecting to the Kubernetes cluster.
+// - installOptions: the installation options including cluster info and chart version.
+// - stackstateConfigs: configuration details for StackState such as the service token and URL.
+// - systemProjectID: the ID of the system project where the chart is deployed.
+// - additionalValues: additional Helm chart values as a map for custom configurations.
+// Returns an error if the chart installation fails or its status cannot be confirmed.
+func InstallStackStateServerChart(client *rancher.Client, installOptions *InstallOptions, systemProjectID string, additionalValues map[string]interface{}) error {
+
```

Comment:
Can we add a RegisterCleanup for the uninstall action as once the test runs, the stackstate chart remains installed.

Reply:
I don't think it does, because the namespace/project is deleted, so it seems like overkill to add another cleanup function.

Reply:
thanks for letting me know. That sounds good as long as the chart is removed.